ledger-api/grpc-definitions/com/daml/ledger/api/v1/event.proto

```diff
@@ -50,14 +52,30 @@ message CreatedEvent {
   // Required
   Identifier template_id = 3;
 
-  // The key of the created contract, if defined.
+  // The key of the created contract.
+  // This will be set if and only if ``create_arguments`` is set and ``template_id`` defines a contract key.
   // Optional
   Value contract_key = 7;
 
   // The arguments that have been used to create the contract.
-  // Required
+  // Set either:
+  //   - if there was a party, which is in the ``witness_parties`` of this event,
+  //     and for which an ``InclusiveFilters`` exists with the ``template_id`` of this event
+  //     among the ``template_ids``,
+  //   - or if there was a party, which is in the ``witness_parties`` of this event,
+  //     and for which a wildcard filter exists (``Filters`` without ``InclusiveFilters``,
+  //     or with an ``InclusiveFilters`` with empty ``template_ids`` and empty ``interface_filters``).
+  // Optional
```

Comment:
Should we specify that this field is serialized according to the serialization rules for the language version belonging to `template_id`?